src/containers/Tablet/Tablet.tsx

```diff
@@ -48,10 +49,14 @@ export const Tablet = () => {
         error,
     } = useTypedSelector((state) => state.tablet);
 
-    const {nodeId: queryNodeId, tenantName: queryTenantName} = parseQuery(location);
-
+    const {
+        nodeId: queryNodeId,
+        tenantName: queryTenantName,
+        type: queryType,
+    } = parseQuery(location);
     const nodeId = tablet.NodeId?.toString() || queryNodeId?.toString();
     const tenantName = tenantPath || queryTenantName?.toString();
+    const type = (tablet.Type?.toString() as EType) || (queryType?.toString() as EType);
```

Comment:
You need parse to string only type from query string, `tablet.Type` already has correct type.
And the type for `queryType` would be `EType | undefined`